hardware_enablement/psap-driver-toolkit.adoc

```diff
@@ -9,7 +9,6 @@ toc::[]
 Learn about the Driver Toolkit and how you can use it as a base image for driver containers for enabling special software and hardware devices on Kubernetes.
 
 :FeatureName: The Driver Toolkit
-include::snippets/technology-preview.adoc[leveloffset=+0]
 
```

Comment:
@slovern This change should to be in a separate PR based off of main.

Reply:
@mburke5678 I think I've fixed that now. The easiest way seemed to be to just add the line back in so there's no change. Does that work for you?